source/backend/api/Solvers/AcquisitionStatusSolver.cs

```diff
@@ -70,25 +70,12 @@ public bool CanEditTakes(AcquisitionStatusTypes? acquisitionStatus)
                 return false;
             }
 
-            bool canEdit;
-            switch (acquisitionStatus)
+            var canEdit = acquisitionStatus switch
             {
-                case AcquisitionStatusTypes.ACTIVE:
-                case AcquisitionStatusTypes.DRAFT:
-                    canEdit = true;
-                    break;
-                case AcquisitionStatusTypes.ARCHIV:
-                case AcquisitionStatusTypes.CANCEL:
-                case AcquisitionStatusTypes.CLOSED:
-                case AcquisitionStatusTypes.COMPLT:
-                case AcquisitionStatusTypes.HOLD:
-                    canEdit = false;
-                    break;
-                default:
-                    canEdit = false;
-                    break;
-            }
-
+                AcquisitionStatusTypes.ACTIVE or AcquisitionStatusTypes.DRAFT => true,
+                AcquisitionStatusTypes.ARCHIV or AcquisitionStatusTypes.CANCEL or AcquisitionStatusTypes.CLOSED or AcquisitionStatusTypes.COMPLT or AcquisitionStatusTypes.HOLD => false,
+                _ => false,
```

Comment:
nit: not sure is more readable

Reply:
I am favor to start introducing more new features.

Reply:
Newer is not necessarily better. If you believe this new style is better, please elaborate. If there is no performance improvements, changes should be aimed at increasing consistency and readability in the code. If new language features help there we should use them, otherwise not.